pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -318,6 +318,9 @@ public static class Broker {
         "pinot.broker.enable.partition.metadata.manager";
     public static final boolean DEFAULT_ENABLE_PARTITION_METADATA_MANAGER = false;
 
+    public static final String CONFIG_OF_MAX_QUERY_RESPONSE_SIZE_BYTES = "pinot.broker.max.query.response.size.bytes";
+    public static final long DEFAULT_MAX_QUERY_RESPONSE_SIZE_BYTES = -1;
```

Comment:
What does -1 mean ?

Reply:
I believe -1 means that there is no limit, but I think it would be more clear and lead to simpler code if we make the default `Long.MAX_VALUE`.

Reply:
Set to Long.MAX_VALUE